lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/QtiImport.java

```diff
@@ -181,7 +181,21 @@ public String guessQuestionType (Node itemnode) {
 		return "Fill In the Blank";
 
 	}
-	
+
+	// response_fin is for numeric response
+	// matching has an explicit type, so just need to see if true false
+	NodeList fin=((Element)itemnode).getElementsByTagName("render_fin");
```

Comment:
I'd use spaces around the = sign here. Easier to read.

Reply:
Solved